vhdbuilder/packer/windows/configure-windows-vhd.ps1

```diff
@@ -320,6 +320,10 @@ function Get-ContainerImages
         Write-Output "* $image"
     }
 
+    # TODO: remove when crictl.exe can find the default config file
```

Comment:
It'd be nice for this comment to have a bit of a discussion about the regression in crictl.